lib/ts/recipe/dashboard/api/listTenants.ts

```diff
@@ -43,6 +48,12 @@ export default async function listTenants(
 ): Promise<Response> {
     let tenantsRes = await Multitenancy.listAllTenants(userContext);
     let finalTenants: TenantListTenantType[] = [];
+    let passwordlessContactMethod: PasswordlessContactMethod | undefined = undefined;
+
+    try {
+        const passwordlessRecipe = PasswordlessRecipe.getInstanceOrThrowError();
```

Comment:
user may have also init thirdpartypasswordless.. that also needs to be taken into account.

Reply:
done